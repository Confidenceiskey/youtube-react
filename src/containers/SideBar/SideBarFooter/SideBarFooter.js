import React from 'react';
import './SideBarFooter.scss';

const SideBarFooter = () => {
  return (
    <React.Fragment>
      <footer className='footer-block'>
        <div>About Press Copyright</div>
        <div>Creators Advertise</div>
        <div>Developers +MyTube</div>
        <div>Legal</div>
      </footer>

      <footer className='footer-block'>
        <div>Terms Privacy</div>
        <div>Policy &amp; Safety</div>
        <div>Test new features</div>
      </footer>

      <footer className='footer-block'>
        <div>All prices include VAT</div>
      </footer>

      <footer className='footer-block'>
        <div>&copy; Producercoder.com - A YouTube clone for educational purposes under fair use.</div>
      </footer>
    </React.Fragment>
  );
}

export default SideBarFooter;
