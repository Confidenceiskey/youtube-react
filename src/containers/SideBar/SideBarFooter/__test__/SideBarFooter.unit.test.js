import React from 'react';
import { shallow } from 'enzyme';
import SideBarFooter from '../SideBarFooter';

it('renders SideBarFooter', () => {
  const wrapper = shallow(
    <SideBarFooter />
  );
  expect(wrapper).toMatchSnapshot();
});
