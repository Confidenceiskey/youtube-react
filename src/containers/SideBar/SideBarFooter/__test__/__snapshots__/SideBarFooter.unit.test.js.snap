// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBarFooter 1`] = `
<Fragment>
  <footer
    className="footer-block"
  >
    <div>
      About Press Copyright
    </div>
    <div>
      Creators Advertise
    </div>
    <div>
      Developers +MyTube
    </div>
    <div>
      Legal
    </div>
  </footer>
  <footer
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new features
    </div>
  </footer>
  <footer
    className="footer-block"
  >
    <div>
      All prices include VAT
    </div>
  </footer>
  <footer
    className="footer-block"
  >
    <div>
      Â© Producercoder.com - A YouTube clone for educational purposes under fair use.
    </div>
  </footer>
</Fragment>
`;
